spec/index.html

```diff
@@ -39,3 +39,18 @@ <h1>Set.prototype.difference(_iterable_)</h1>
     <h1>Set.prototype.symmetricDifference(_iterable_)</h1>
     <emu-import href="./set-prototype-symmetric-difference.html"></emu-import>
 </emu-clause>
+
+<emu-clause id="Set.prototype.isSubsetOf">
+    <h1>Set.prototype.isSubsetOf(_iterable_)</h1>
+    <emu-import href="./set-prototype-is-subset-of.html"></emu-import>
+</emu-clause>
+
+<emu-clause id="Set.prototype.isSupersetOf">
+    <h1>Set.prototype.isSupersetOf(_iterable_)</h1>
+    <emu-import href="./set-prototype-is-superset-of.html"></emu-import>
+</emu-clause>
+
+<emu-clause id="Set.prototype.isDisjointWith">
```

Comment:
Small note: I believe "is disjoint from" is significantly more common than "is disjoint with". Googling each phrase gives almost an order of magnitude more results for "is disjoint from", and the first result for "is disjoint with" is someone asking if it's correct (whereas the first result for "is disjoint from" is a math article).

Reply:
Can you raise new issue about this?